src/lib/api/contrib/invenio/InvenioRequestSerializer.js

```diff
@@ -82,6 +83,9 @@ export class InvenioRequestSerializer {
     if (size > 0) {
       getParams['size'] = size;
     }
+    if (!_isEmpty(hiddenParams)) {
+      getParams = {...getParams, ...hiddenParams};
```

Comment:
I think `hiddenParams` has to pass through `this._addFilters(hiddenParams);`, to get translated to proper querystring parameters.